frontend/spec/controllers/spree/checkout_controller_spec.rb

```diff
@@ -144,25 +144,29 @@ def post_address
         end
 
         context "with a billing and shipping address" do
-          before do
-            @expected_bill_address_id = order.bill_address.id
-            @expected_ship_address_id = order.ship_address.id
-
-            post :update, {
-                state: "address",
-                order: {
-                    bill_address_attributes: order.bill_address.attributes.except("created_at", "updated_at"),
-                    ship_address_attributes: order.ship_address.attributes.except("created_at", "updated_at"),
-                    use_billing: false
-                }
+          subject do
+            -> {
```

Comment:
Is this lamda necessary? I think the `subject` call will already make the code in the block callable?